packages/graphql/src/graphql/input-objects/AsDuplicate.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GraphQLBoolean } from "graphql";
+
+export const AsDuplicate = {
+    type: GraphQLBoolean,
+    description:
+        "Whether or not to create a duplicate of relationship if it already exists, instead of just updating any properties.",
+};
```

Comment:
I would perhaps move this from the `graphql` directory - it's not technically a GraphQL type, so I would move it into something like a `fields` directory within the `schema` directory. I would also export it as `asDuplicate` so it can be used without reassignment.